src/Controller/EnMarche/ReferentController.php

```diff
@@ -144,4 +151,56 @@ public function commiteesAction(): Response
             'managedCommitteesJson' => $exporter->exportAsJson($list),
         ]);
     }
+
+    /**
+     * @Route("/organigrame", name="app_referent_organizational_chart")
```

Comment:
`organigramme`

Reply:
in english it's organizational chart no ?

Reply:
Yes

Reply:
OOoooo ok, I didn't understand it was for the path ^^,